daily_practice/challenges/classes-challenges.test.js

```diff
@@ -0,0 +1,49 @@
+// Using the Coffee Maker Class, write tests and complete the challenges
+class Coffee {
+  constructor(type, cream, sugar){
+    this.type = type.toLowerCase()
+    this.cream = cream
+    this.sugar = sugar
+  }
+
+  coffeeProfile(){
+    return(`${this.type}: ${this.creams()}, ${this.sugars()}`)
+  }
+
+  creams(){
+    if (this.cream > 1){
+      return `${this.cream} creams`
+    } else {
+      return `${this.cream} cream`
+    }
+  }
+
+  sugars(){
+    if (this.sugar > 1){
+      return `${this.sugar} sugars`
+    } else {
+      return `${this.sugar} sugar`
+    }
+  }
+}
+// Write the code that makes a black coffee.
+var black = new Coffee()
+
+console.log(black.coffeeProfile(0))
```

Comment:
We shouldn't see console statements here, we should see an assertion via your test framework.